### PR TITLE
Draft ietf lpwan schc over lorawan 14 - Following IESG review

### PR DESCRIPTION
**Pull request by Oliv4945**
Hi @ivajloip @jca-klk 

Here is a new (and maybe last MR before RFC editors) including all IESG directors comments. Would you mind to review it?

Thanks

### CODE REVIEW

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 1]
```diff
@@ -507,40 +521,44 @@ the fragment receiver. A single fragmentation rule is defined.
 SCHC F/R MUST concatenate FPort and LoRaWAN payload to retrieve the SCHC
 Packet, as per {{lorawan-schc-payload}}.
 
-* SCHC header size is two bytes (the FPort byte + 1 additional byte).
-* RuleID: 8 bits stored in LoRaWAN FPort.
 * SCHC fragmentation reliability mode: `ACK-on-Error`.
-* DTag: Size is 0 bit, not used.
+* SCHC header size is two bytes (the FPort byte + 1 additional byte).
+* RuleID: 8 bits stored in LoRaWAN FPort. cf {{rule-id-management}}
+* DTag: Size T=0 bit, not used. cf {{DTag}}, §8.2.3. Integrity Checking
```
Comment (jca-klk) [inline draft-ietf-lpwan-schc-over-lorawan.md]: Not sure §8.2.3 is what you intended. Isn't `{{DTag}}` sufficient?

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 2]
```diff
@@ -664,17 +681,20 @@ Packet as described in {{lorawan-schc-payload}}.
...
-* RuleID: 8 bits stored in LoRaWAN FPort.
-* Window index (unicast only): encoded on W=1 bit, as per [RFC8724].
-* DTag: Size is 0 bit, not used.
+* RuleID: 8 bits stored in LoRaWAN FPort. cf {{rule-id-management}}
+* DTag: Size T=0 bit, not used. cf {{DTag}}
 * FCN: The FCN field is encoded on N=1 bit, so WINDOW_SIZE = 1 tile.
-* RCS: Use recommended calculation algorithm in [RFC8724].
-* MAX_ACK_REQUESTS: 8.
-* Retransmission timer:  See {{downlink-retransmission-timer}}.
+* RCS: Use recommended calculation algorithm in [RFC8724], §8.2.3. Integrity Checking.
```
Comment (jca-klk) [inline draft-ietf-lpwan-schc-over-lorawan.md]: Same here, is §8.2.3 intended?

File: draft-ietf-lpwan-schc-over-lorawan.md [hunk 3]
```diff
@@ -81,7 +81,8 @@ and modes of operation need to be set appropriately for each of the LPWAN
 technologies.
 
 This document describes the parameters and modes of operation when
-SCHC is used over LoRaWAN networks.
+SCHC is used over LoRaWAN networks. LoRaWAN protocol is specified by the
+LoRa® Alliance in {{lora-alliance-spec}}
```
Comment (jca-klk) [inline draft-ietf-lpwan-schc-over-lorawan.md:85]: LoRa Alliance should be noted `LoRa Alliance®` 
See https://lora-alliance.org/ for public references

Comment (jca-klk) [reply draft-ietf-lpwan-schc-over-lorawan.md:85]: Not sure RFCs need to follow those (r) and (tm) rules though

**Review by jca-klk — APPROVED** [review]

File: draft-ietf-lpwan-schc-over-lorawan.md (hunk 1 above)
Comment (Oliv4945) [reply draft-ietf-lpwan-schc-over-lorawan.md]: Thanks, fixed in dc7f630

File: draft-ietf-lpwan-schc-over-lorawan.md (hunk 2 above)
Comment (Oliv4945) [reply draft-ietf-lpwan-schc-over-lorawan.md]: Yes, it is the chapter of the RFC8724 where the RCS is recommended.
It was the advice from B. Kaduk, I slightly changed it to:

```
RCS: Use recommended calculation algorithm in [RFC8724] (§8.2.3. Integrity Checking).
```

Is it more explicit ?

File: draft-ietf-lpwan-schc-over-lorawan.md (hunk 3 above)
Comment (Oliv4945) [reply draft-ietf-lpwan-schc-over-lorawan.md:85]: Thanks, corrected. I checked with Olivier S. and he advised me to put the registered for the first apparition of LoRa, LoRaWAN and LoRa Alliance. Not sure about IETF rule though, I'll ask the editors.

**Review by ivajloip — APPROVED** [review]
Looks good to me, thank you for those changes!

**Pull request merged**